Dockerfile

```diff
@@ -1,4 +1,26 @@
-FROM phusion/baseimage:jammy-1.0.1
+FROM ubuntu:jammy as builder
+
+WORKDIR /root
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+        git ca-certificates
+
+RUN git clone --branch master --single-branch https://github.com/phusion/baseimage-docker.git && \
+    git clone --branch master --single-branch https://github.com/koreader/koreader-sync-server.git
```

Comment:
But this doesn't make sense to me. The point of a local clone is that it's a local clone, not to make it a game of trying to figure out why your local changes aren't being found. ;-)

Reply:
The point of this was to download the latest files from GitHub, and then later those files would get copied into the Kosync image. But as you mentioned, downloading the latest files from GitHub probably isn't the best choice anyway.